apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsDateControl.tsx

```diff
@@ -33,7 +33,7 @@ export function JsonFormsDateControl({
         <FormLabel description={description}>{label}</FormLabel>
         <DatePicker
           inputValue={!!data ? String(data) : undefined}
-          allowManualInput={false}
+          allowManualInput={true}
```

Comment:
note: this was set to true to allow users to remove the value from the date picker, as there is no "clear input" functionality in the OGP design system datepicker